### PR TITLE
Portal library: index param in lib functions must be optional #9843

### PR DESCRIPTION
**Pull request by edloidas**

### CODE REVIEW

File: modules/lib/lib-portal/src/main/resources/lib/xp/portal.ts [hunk 1]
```diff
@@ -526,7 +526,7 @@ export function getMultipartForm(): MultipartForm {
  *
  * @returns {object|null} The named multipart form item.
  */
-export function getMultipartItem(name: string, index: number): MultipartItem | null {
+export function getMultipartItem(name: string, index?: number): MultipartItem | null {
     const bean = __.newBean<MultipartHandler>('com.enonic.xp.lib.portal.multipart.MultipartHandler');
     return __.toNativeObject(bean.getItem(name, index ?? 0));
 }
```
Comment (tajakobsen) [inline modules/lib/lib-portal/src/main/resources/lib/xp/portal.ts:532]: I prefer this syntax, as it ties the default value to the variable to the `index` variable.

```suggestion
export function getMultipartItem(name: string, index: number = 0): MultipartItem | null {
    const bean = __.newBean<MultipartHandler>('com.enonic.xp.lib.portal.multipart.MultipartHandler');
    return __.toNativeObject(bean.getItem(name, index));
}
```

**Review by tajakobsen — CHANGES_REQUESTED** [review]
I just have a suggestion of an alternative syntax. Nothing blocking.

File: modules/lib/lib-portal/src/main/resources/lib/xp/portal.ts (hunk 1 above)
Comment (edloidas) [reply modules/lib/lib-portal/src/main/resources/lib/xp/portal.ts]: Yeah, the default value will look better.

**Review by tajakobsen — APPROVED** [review]
Looks good.

**Pull request merged**